keps/sig-auth/3130-kms-observability/README.md

```diff
@@ -134,16 +135,18 @@ Integration test covering:
 Pick one of these and delete the rest.
 -->
 
-- [x] Other
-  - Describe the mechanism: The UID field will be added by default to the `EncryptRequest` and `DecryptRequest` structs in the kms-plugin interface. For older Kubernetes versions where UID isn't generated, the UID field will be empty.
+- Feature gate
+  - Feature gate name: `KMSOperationUID` (enabled by default)
```

Comment:
I think the specifics are:

```go
FeatureSpec{
	Default: true,  // enabled by default
	LockToDefault: false, // can be disabled if needed
	PreRelease: featuregate.Deprecated, // gate will dropped in 1.25
}
```

Reply:
Or maybe `PreRelease: featuregate.GA` is the correct thing?  I think this field refers to the feature itself and not the gate?

Reply:
Yeah, `featuregate.GA` seems to refer to the feature, so using that